.gitignore

```diff
@@ -0,0 +1,5 @@
+# macOS
```

Comment:
This file doesn't belong in the repository, instead it should go in your system git configuration.

https://git-scm.com/docs/gitignore

Reply:
Thanks. I wasn't aware of that subtle difference.